packages/astro/test/fixtures/ssr-redirect/src/pages/secret.astro

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
Just a callout on test fixtures going forward: due to a new `noExternal` dependency in Vite 3, we'll need to include a `package.json` with `astro` as a dependency for every test suite. Otherwise, Vite will fail to pre-bundle Astro. I'd recommend adding a `package.json` for this new suite as well!

Reply:
Thanks, added one.